samples/pmic/native/npm1300_one_button/README.rst

```diff
@@ -0,0 +1,120 @@
+.. _npm1300_one_button:
+
+nPM1300: One button
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The One button sample demonstrates how to support wake-up, shutdown and user interactions through
+a single button connected to the nPM1300.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires an nPM1300 EK.
+
+Overview
+********
+
+The sample uses a single button for controlling the device state.
+The sample controls an LED, load switch and the PMIC power mode based on button press duration.
+
+Wiring
+******
+
+#. Make the following connections on the nPM1300 EK:
+
+   * Remove all existing connections.
+   * Connect a suitable battery to the **J2** connector.
+   * On the **P17** pin header, connect **HOST** and **LED2** pins with a jumper.
+   * On the **P13** pin header, connect **RSET1** and **VSET1** pins with a jumper.
+   * On the **P14** pin header, connect **RSET2** and **VSET2** pins with a jumper.
+   * On the **LOAD SWITCH CONFIG** pin header, connect **VOUT2** and **LSIN2** pins with a jumper.
+
```

Comment:
I recommend we use LS1 instead, the initial EKs are using LS2 as a workaround for an issue on EK v0.9.2, so LS2 will not be available.
We can also use the onboard high load easier this way:
VOUT1 - LSIN1
HIGH - LSOUT2

Reply:
The code is actually using LDSW1, but the wiring info is wrong, I will update it.